src/subscriptions-view.jsx

```diff
@@ -390,18 +387,15 @@ class SubscriptionsView extends React.Component {
         let message;
 
         if (status === "service-unavailable") {
-            message = _("The rhsm service is unavailable. Make sure subscription-manager is installed \
-and try reloading the page. Additionally, make sure that you have checked the \
-'Reuse my password for privileged tasks' checkbox on the login page.");
+            message = _("The rhsm service is unavailable. Make sure subscription-manager is installed and try reloading the page. Additionally, make sure that you have checked the 'Reuse my password for privileged tasks' checkbox on the login page.");
```

Comment:
This is very long line. Can we maybe use here multiline string connected with `+` (as shown in https://eslint.org/docs/latest/rules/no-multi-str)?

Reply:
Yes, the line is very long unfortunately. This is how we deal with long string in cockpit too ([similar example here](https://github.com/cockpit-project/cockpit/blob/main/pkg/systemd/overview-cards/realmd.jsx#L320)). 
I'm not sure if using the syntax with `+` could mess something up with translations. Probably depends on ngettext and I dont think it interprets JS.

Otherwise I can revert this change and tell eslint to ignore this part. What do you think?